src/index.ts

```diff
@@ -36,7 +38,11 @@ export {
   SessionProcessor,
   User,
   Session,
-  Password
+  Password,
+  // Addons
```

Comment:
comment?

Reply:
It's a valid comment, not a commented import.